canonical-web-and-design/ubuntu.com
# Changed 20.04 to 18.04 and updated links

## Done

- Changed 20.04 to 18.04 and updated links

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare the changes against the [copy doc](https://docs.google.com/document/d/1GUlSATrqZmGrsk4DhP9W0s-gcOuFxpdKVgx7YE4g780/edit?ts=602ea235#) and accept the correct ones.


## Issue / Card

Fixes [#3839](https://github.com/canonical-web-and-design/web-squad/issues/3839)
